src/Cake/Features/Bootstrapping/BootstrapFeature.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Linq;
+using Autofac;
+using Cake.Core;
+using Cake.Core.Diagnostics;
+using Cake.Core.IO;
+using Cake.Core.Scripting;
+using Cake.Core.Scripting.Analysis;
+using Cake.Infrastructure;
+using Spectre.Cli;
+
+namespace Cake.Features.Bootstrapping
+{
+    public interface IBootstrapFeature
+    {
+        int Run(IRemainingArguments arguments, BootstrapFeatureSettings settings);
+    }
+
+    public sealed class BootstrapFeature : Feature, IBootstrapFeature
+    {
+        private readonly ICakeEnvironment _environment;
+
+        public BootstrapFeature(
+            IFileSystem fileSystem,
+            ICakeEnvironment environment,
+            IContainerBootstrapper bootstrapper) : base(fileSystem, environment, bootstrapper)
+        {
+            _environment = environment;
+        }
+
+        public int Run(IRemainingArguments arguments, BootstrapFeatureSettings settings)
+        {
+            // Fix the script path.
+            settings.Script = settings.Script ?? new FilePath("build.cake");
```

Comment:
maybe move the `new FilePath("build.cake")` to a constant or set a default value to the `BootstrapFeatureSettings .Script` property?